libdnf/module/modulemd/ModuleProfile.cpp

```diff
@@ -32,15 +32,17 @@ std::string ModuleProfile::getName() const
     if (!profile) {
         return {};
     }
-    return modulemd_profile_peek_name(profile);
+    auto name = modulemd_profile_peek_name(profile);
+    return name ? name : "";
```

Comment:
Personally I'd do that like this:
```cpp
if (auto name = modulemd_profile_peek_name(profile))
  return name;
return {};
```

but either way, this patch looks like a useful fix.